helix-core/src/test/java/org/apache/helix/manager/zk/TestZkBaseDataAccessor.java

```diff
@@ -146,7 +157,7 @@ public void testSyncDoSet() {
     ZNRecord record = new ZNRecord("submsg_0");
     ZkBaseDataAccessor<ZNRecord> accessor = new ZkBaseDataAccessor<ZNRecord>(_gZkClient);
 
-    AccessResult result = accessor.doSet(path, record, -1, AccessOption.PERSISTENT);
+    AccessResult result = accessor.doSet(_gZkClient, path, record, -1, AccessOption.PERSISTENT);
```

Comment:
If we keep the old public API instead of removing, we shouldn't need to change this at all. In other words, we shouldn't see any diffs here.